src/app/mod.rs

```diff
@@ -335,16 +336,15 @@ impl Surreal {
 			},
 		};
 		let num_statements = response.num_statements();
-		let response = if num_statements > 1 {
+		let response = {
 			let mut output = Vec::<Value>::with_capacity(num_statements);
 			for index in 0..num_statements {
-				output.push(response.take(index)?);
+				let v: Value = response.take(index)?;
```

Comment:
```suggestion
				let v = response.take(index)?;
```

The type hint shouldn't be necessary here. They compiler should be able to infer it from `Vec<Value>`.